base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -171,6 +171,8 @@ where
         let mut base_node_service_event_stream = self.base_node_service.get_event_stream();
 
         debug!(target: LOG_TARGET, "Output Manager Service started");
+        // Outputs marked as shorttermencumbered are not yet stored as transactions in the TMS, so lets clear them
+        self.resources.db.clear_short_term_encumberances()?;
```

Comment:
It feels like this could create problems, for example, we create an interactive transaction, send the initial request to the receiver and quit immediately afterwards. Then we start up again and clear all those encumbered outputs, make a new transaction and use them again, and so on.

Reply:
as soon as we send them out, we flag them as encumbered, not short_term_encumbered.

Reply:
Perfect